go/test/endtoend/transaction/twopc/main_test.go

```diff
@@ -126,6 +127,8 @@ func cleanup(t *testing.T) {
 	defer conn.Close()
 
 	_, _ = utils.ExecAllowError(t, conn, "delete from twopc_user")
+	_, _ = utils.ExecAllowError(t, conn, "delete from twopc_fuzzer_insert")
+	_, _ = utils.ExecAllowError(t, conn, "delete from twopc_fuzzer_update")
 }
 
```

Comment:
this could fail when it exceeds the 10k limit and will continue to fail next test if they use the same table.

Reply:
Good point, let me fix it up.

Reply:
Fixed up.